sdk/attestation/azure-security-attestation/azure/security/attestation/_models.py

```diff
@@ -69,242 +79,244 @@ class PolicyCertificatesModificationResult(object):
-        :paramtype policy_claims: dict
-        :keyword verifier_type: Verifier which generated this token.
-        :paramtype verifier_type: str
-        :keyword policy_signer: If the policy which processed the request is signed, 
-            this will be the certificate which signed the policy.
-        :paramtype policy_signer: azure.security.attestation.AttestationSigner
-        :keyword policy_hash: The hash of the policy which processed the attestation 
-            evidence.
-        :paramtype policy_hash: str
-        :keyword is_debuggable: True if the SGX enclave being attested is debuggable.
-        :paramtype is_debuggable: bool
-        :keyword product_id: Product ID for the SGX enclave being attested.
-        :paramtype product_id: int
-        :keyword mr_enclave: MRENCLAVE value for the SGX enclave being attested.
-        :paramtype mr_enclave: str
-        :keyword mr_signer: MRSIGNER value for the SGX enclave being attested.
-        :paramtype mr_signer: str
-        :keyword svn: Security version number for the SGX enclave being attested.
-        :paramtype svn: int
-        :keyword sgx_collateral: Collateral which identifies the collateral used to 
-            create the token.
-        :paramtype sgx_collateral: dict
-    
-        """
-        self._issuer = kwargs.pop("issuer", None) #type:Union[str, None]
-        self._confirmation = kwargs.pop("confirmation", None) #type:Union[dict, None]
-        self._unique_identifier = kwargs.pop("unique_identifier", None) #type:Union[str, None]
-        self._nonce = kwargs.pop("nonce", None) #type:Union[str, None]
-        self._version = kwargs.pop("version", None) #type:Union[str, None]
-        self._runtime_claims = kwargs.pop("runtime_claims", None) #type:Union[dict, None]
-        self._inittime_claims = kwargs.pop("inittime_claims", None) #type:Union[dict, None]
-        self._policy_claims = kwargs.pop("policy_claims", None) #type:Union[dict, None]
-        self._verifier_type = kwargs.pop("verifier_type", None) #type:Union[str. None]
-        self._policy_signer = kwargs.pop("policy_signer", None) #type:Union[AttestationSigner, None]
-        self._policy_hash = kwargs.pop("policy_hash", None) #type:Union[str, None]
-        self._is_debuggable = kwargs.pop("is_debuggable", None) #type:Union[bool, None]
-        self._product_id = kwargs.pop("product_id", None) #type:Union[int, None]
-        self._mr_enclave = kwargs.pop("mr_enclave", None) #type:Union[str, None]
-        self._mr_signer = kwargs.pop("mr_signer", None) #type:Union[str, None]
-        self._svn = kwargs.pop("svn", None) #type:Union[int, None]
-        self._enclave_held_data = kwargs.pop("enclave_held_data", None) #type:Union[bytes, None]
-        self._sgx_collateral = kwargs.pop("sgx_collateral", None) #type:Union[dict, None]
+        # type: (Dict[str,Any]) -> None
+        self._issuer = kwargs.pop("issuer")  # type: Union[str, None]
```

Comment:
Union[str, None] and Optional[str] are functionally equivalent; I have a slight preference for the latter, but I've made enough nitpicking comments already 🤠
```suggestion
        self._issuer = kwargs.pop("issuer")  # type: Optional[str]
```

Reply:
There are never too many nitpicking comments. But Anna had suggested Union[xxx, None] in API review feedback, which is why I went with that.